app/routes/funding/models/what-funding.js

```diff
@@ -9,12 +11,19 @@ function ViewModel (eligibleFunding, selected, error) {
         classes: 'govuk-fieldset__legend--l'
       }
     },
-    items: mapStandards(eligibleFunding, selected)
+    hint: {
+      text: 'Choose all that apply.'
+    },
+    items: mapStandards(eligibleFunding, selected).map(funding => { funding.checked = isChecked(selected, funding.value); return funding })
   }
 
   if (error) {
-    this.model.errorMessage = {
-      text: 'Please choose the funding you would like to apply for '
+    this.model.error = {
+      errorList: [
+        {
+          text: 'Select an option'
```

Comment:
Maybe change the text for the error to 

Please choose the funding you would like to apply for.

Also to make it consistent, add "There is a problem" title. 

```
this.model.error = {
      titleText: "There is a problem",
      errorList: [
        {
          text: 'Select an option'
```